application/config.json.template

```diff
@@ -1,6 +1,6 @@
 {
     "token": "<put_your_token_here>",
-    "gistApiKey": "<your_gist_personal_access_token>",
+    "githubApiKey": "<your_github_personal_access_token>",
```

Comment:
note that renames like that are causing quite some confusion among other users who dont follow the changes regularly.

its fine though, but i want to raise awareness

Reply:
i saw your comment regarding this in old PR, i understand but i think `githubApiKey` does sound more accurate with this now.

Reply:
yeah. its a fine change.